docs/src/content/docs/challenges/rxjs/38-rxjs-catch-error.md

```diff
@@ -1,25 +1,30 @@
 ---
 title: 🟢 catchError
-description: Challenge 38 is about learning error handling with catchError rxjs operator
+description: Challenge 38 is about learning obervable completion.
```

Comment:
You need to rebase your branch to the latest commit
The doc has changed
- you need to add some properties (look at another doc)
- delete l.9 
- delete all the footer